### PR TITLE
Add a function to load the labeled json data

### PR DESCRIPTION
**Pull request by zhangk1551**
Provide a function to directly read the json data besides yaml files.

### CODE REVIEW

**Comment by zhangk1551** [comment]
an example (the static car in the center is marked as parked):

https://drive.google.com/file/d/1v2M_WAD02MnkVznO_ORgkRfRA4wDhbS6/view?usp=sharing

**Comment by ruinixu** [comment]
I can't see the example, just edit the sharing permissions on the link

**Comment by adscib** [comment]
Could you add a section to the README describing how to add a custom scenario using this format so that I can test it?

**Comment by ruinixu** [comment]
> Could you add a section to the README describing how to add a custom scenario using this format so that I can test it?

I also have the branch `custom-scenario-readme` on this repo for scenario-editor-specific instructions, feel free to merge in your README, theres a TODO in there you can continue off of

**Comment by zhangk1551** [comment]
@ruinixu Thanks. The readme looks good. I merged it to this branch and also addressed the TODO.

File: README.md [hunk 1]
```diff
@@ -22,6 +22,23 @@ TorchDriveEnv comes bundled with a predefined collection of diverse and challeng
...
+    * Cars in the scenario can be marked as stationary vehicles by selecting them on the map, then selecting `Mark as Parked`.
+    * If you would like a car to follow a certain trajectory, first select the car, then select `Add Waypoints` to define a waypoint sequence for the car to follow. To end a sequence, click on the last set waypoint.
+    * If you would like to validate your waypoints and initial conditions, you can click on `DRIVE` to run an example of what your simulation might look like, and adjust the scenario as needed.
+
+* Define your ego vehicle initialization range and waypoints with `Start Ego Waypoint Sequence`. The first two points define the range where the starting position is sampled from. To end a sequence, click on the last set waypoint.
+
+* Export your scenario JSON files to the same directory and use [load_labeled_data(data_dir)](https://github.com/inverted-ai/torchdriveenv/blob/master/torchdriveenv/env_utils.py#L31) to load the scenarios.
```
Comment (adscib) [inline README.md:40]: Is there any way to make a script that would take a path to the JSON file, construct the environment, and display something that shows that the scenario was loaded correctly? It's fine to use some dummy policy like a zero action.

Comment (zhangk1551) [reply README.md:40]: image
You could load a JSON file via the labeling tool.

Comment (adscib) [reply README.md:40]: Yes, but I'd like to construct an environment and see that it's playing the scenario. That would provide a much easier entry point than having to figure out where the `load_labeled_data` function needs to be called and how to see that it's executing the right scenario.

Comment (zhangk1551) [reply README.md:40]: I see. You could try: https://github.com/inverted-ai/torchdriveenv/blob/master/examples/waypoint_suite_env_example.ipynb, replace the `load_default_validation_data()` with `load_labeled_data`. That's how I generate the video in this PR.

Comment (adscib) [reply README.md:40]: This notebook doesn't run. It seems that `load_default_env_config` no longer exists.

**Comment by zhangk1551** [comment]
A screen recording to label a case: https://www.loom.com/share/2c359cb3128947c4b20ac5697ccd0571?sid=6c970927-49a2-453f-8a14-b4d6ec59f3d4 (The video is tempory. I can't download  because I don't have a membership.)

The json file: 
[carla_Town07_3way_scenario (2).json](https://github.com/user-attachments/files/17348067/carla_Town07_3way_scenario.2.json)
@ruinixu It seems that I can't import the json file I export to the labelling tool, could you try it on your side?


The corresponding video rendered by the notebook: 

https://drive.google.com/file/d/1v2M_WAD02MnkVznO_ORgkRfRA4wDhbS6/view?usp=sharing

**Review by adscib — APPROVED** [review]
Looks good! I was able to run it with my own scenario.

**Pull request merged**